packages/ui/src/components/shared/SettingsWalletButton.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import DarkModeIcon from '@mui/icons-material/DarkMode';
+import LightModeIcon from '@mui/icons-material/LightMode';
+import SettingsIcon from '@mui/icons-material/Settings';
+import SettingsBrightnessIcon from '@mui/icons-material/SettingsBrightness';
+import { Button, ButtonGroup, Dialog, DialogContent, DialogContentText, IconButton } from '@mui/material';
+import DialogTitle from 'components/shared/DialogTitle';
+import { settingActions, ThemeMode } from 'redux/slices/settings';
+import { RootState } from 'redux/store';
+import { Props } from 'types';
+
+const SettingsWalletButton: FC<Props> = () => {
+  const [open, setOpen] = useState(false);
+  const dispatch = useDispatch();
+  const { themeMode } = useSelector((state: RootState) => state.settings);
+  const { seedReady, locked } = useSelector((state: RootState) => state.app);
+
+  if (!seedReady || locked) {
+    return null;
+  }
+
+  const handleClose = () => setOpen(false);
+  const switchThemeMode = (mode: ThemeMode) => {
+    dispatch(settingActions.switchThemeMode(mode));
+  };
+
+  return (
+    <>
+      <IconButton size='small' title='Open settings' onClick={() => setOpen(true)}>
+        <SettingsIcon />
+      </IconButton>
+      <Dialog open={open} onClose={handleClose} maxWidth='sm' fullWidth>
+        <DialogTitle onClose={handleClose}>Settings</DialogTitle>
+        <DialogContent>
+          <DialogContentText>Themes</DialogContentText>
+          <ButtonGroup className='' fullWidth>
+            <Button
+              variant={themeMode == ThemeMode.Dark ? 'contained' : 'outlined'}
+              onClick={() => switchThemeMode(ThemeMode.Dark)}>
+              <DarkModeIcon className='px-0.5' />
```

Comment:
Add more space to this as well:
```suggestion
              <DarkModeIcon className='mr-2' fontSize='small'/>
```

Same for other places as well.